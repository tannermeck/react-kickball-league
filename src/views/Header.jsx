import { NavLink } from 'react-router-dom'

function Header(){
    return (
        <>
            <NavLink to='/'>Home</NavLink>{' '}
            <NavLink to='/teams'>Teams</NavLink>{' '}
            
        </>
    )
}
export default Header;