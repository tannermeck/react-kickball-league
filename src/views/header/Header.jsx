import { NavLink } from 'react-router-dom'
import './header.css'

function Header(){
    return (
        <div className='headerContainer'>
            <NavLink className='header' to='/'>Home</NavLink>{' '}
            <NavLink className='header' to='/teams'>Teams</NavLink>{' '}
            <NavLink className='header' to='/players'>Players</NavLink>
        </div>
    )
}
export default Header;