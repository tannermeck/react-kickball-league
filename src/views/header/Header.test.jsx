import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router';
import Header from './Header';

it('should render the header', () => {

    const { container } = render(
                            <MemoryRouter>
                                <Header />
                            </MemoryRouter>)
    expect(container).toMatchSnapshot();
})