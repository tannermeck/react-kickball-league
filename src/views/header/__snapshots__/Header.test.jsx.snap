// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the header 1`] = `
<div>
  <div
    class="headerContainer"
  >
    <a
      aria-current="page"
      class="header active"
      href="/"
    >
      Home
    </a>
     
    <a
      class="header"
      href="/teams"
    >
      Teams
    </a>
     
    <a
      class="header"
      href="/players"
    >
      Players
    </a>
  </div>
</div>
`;
